src/app/clusters/level-control/level-control.cpp

```diff
@@ -650,16 +652,34 @@ static void moveHandler(uint8_t commandId, uint8_t moveMode, uint8_t rate, uint8
         }
     }
 
-    // If the Rate field is 0xFF, the device should move as fast as it is able.
-    // Otherwise, the rate is in units per second.
+    // If the Rate field is 0xFF, the device should move at the default move rate, if available,
+    // Otherwise, move as fast as possible
     if (rate == 0xFF)
     {
-        state->eventDurationMs = FASTEST_TRANSITION_TIME_MS;
+        uint16_t defaultMoveRate;
+        status = emberAfReadServerAttribute(endpoint, ZCL_LEVEL_CONTROL_CLUSTER_ID, ZCL_DEFAULT_MOVE_RATE_ATTRIBUTE_ID,
+                                            (uint8_t *) &defaultMoveRate, sizeof(defaultMoveRate));
+        if (status != EMBER_ZCL_STATUS_SUCCESS)
+        {
+            emberAfLevelControlClusterPrintln("ERR: reading default move rate %x", status);
+            state->eventDurationMs = FASTEST_TRANSITION_TIME_MS;
+        }
+        else
+        {
+            // nonsensical case, means "don't move", so we're done
```

Comment:
Again, this should be defined in the CHIP data model....  Can probably be part of the issue I asked for above.